haskell-src/Concordium/Types/ProtocolLevelTokens/CBOR.hs

```diff
@@ -220,6 +221,8 @@ data TokenInitializationParameters = TokenInitializationParameters
       tipName :: !Text,
       -- | A URL pointing to the token metadata.
       tipMetadata :: !Text,
+      -- | The governance account of this token.
+      tipGovernanceAccount :: AccountAddress,
```

Comment:
maybe `tipIssuer` is a better name for the field.

Reply:
I feel like maybe the CBOR encoding should be the same, but is it not weird to have an abstract "holder" as the governance account? As an example, how would a contract address ever govern the token?

Reply:
Changed to `TokenHolder`